client/settings/connect-stripe-account/index.js

```diff
@@ -58,9 +55,9 @@ const ButtonWrapper = styled.div`
 const ConnectStripeAccount = () => {
 	const [ hasError, setHasError ] = useState( false );
 
-	const handleErrorChange = ( error ) => {
+	const handleErrorChange = useCallback( ( error ) => {
 		setHasError( !! error );
-	};
+	}, [] );
```

Comment:
Nit: should this not depend on `setHasError`?
```suggestion
	}, [ setHasError ] );
```